includes/class-create-block-theme-api.php

```diff
@@ -372,6 +382,9 @@ function rest_export_child_cloned_theme( $request ) {
 		}
 
 		$zip->close();
+		wp_cache_flush();
+
+		wp_cache_flush();
```

Comment:
I think this is a typo - I'm guessing we don't need to flush the cache twice here 👀

Reply:
haha, true. Fixed.